README.md

```diff
@@ -1,25 +1,30 @@
-# @api3/chains
-
-> The single source of truth for chain-related information
-
-- [Notes](#notes)
```

Comment:
The readme is small enough that a table of contents isn't really necessary imo

Reply:
Agreed